### ISSUE TITLE
Error trying to download new set of images 

### ISSUE THREAD

**Issue by JPedro18** [kvos/CoastSat#554]
While downloading a set of images for a specific site, I encountered an error message, likely due to a lost internet connection, stating that I was not signed up for Earth Engine. To resolve this, I attempted to restart the process from where it had stopped by selecting only the remaining satellite datasets. However, when I retried the command SDS_download.get_metadata(inputs), I received another error. This error seems to be related to the sitename, as changing the sitename allows the process to work correctly.

Below are the details and relevant screenshots for reference:

1- Settings for the first attempt:
![image](https://github.com/user-attachments/assets/6a9933b4-4b4b-4ba0-b309-6988c4425072)

2- Error received during download:
![image](https://github.com/user-attachments/assets/e52deb1d-181b-4d6b-965b-4368cbee7f5b)

3- Settings for the second attempt:
![image](https://github.com/user-attachments/assets/486691a7-3941-4747-b42d-9f0830b79aa6)

4- This is the error I receive when I try **metadata = SDS_download.retrieve_images(inputs)**  or **SDS_download.check_images_available(inputs)** again:
![image](https://github.com/user-attachments/assets/7d29c000-cd04-4731-8049-36cf4c971f16)

As mentioned earlier, if I change the sitename, the process works just fine.

**Comment by JPedro18** [comment]
I just noticed that I used a special character in the sitename, which might be causing the issue. However, it is worth noting that it worked correctly during the first attempt.

**Comment by kvos** [comment]
yes that's an issue with pandas not reading the character. Only use utf-8 encoding and you should be fine.
